src/js/components/VolumeDetailsComponent.jsx

```diff
@@ -5,6 +5,45 @@ export default React.createClass({
   propTypes: {
     volume: React.PropTypes.object
   },
+  getTask: function (volume) {
+    if (volume.taskId == null) {
+      return null;
+    }
+    return (
+      <div>
+        <dt>Task Id</dt>
+        <dd>
+          <a href={volume.taskURI}>{volume.taskId}</a>
```

Comment:
Are we guaranteed to always receive these 2 properties? At least we should ensure that if `taskURI` is not there this is not a link.